tenantvariables/src/main/java/org/assimbly/tenantvariables/TenantVariablesProcessor.java

```diff
@@ -1,12 +1,13 @@
 package org.assimbly.tenantvariables;
```

Comment:
Just a question. As we will drop this legacy encryption in 4.16.x. Is it then good to keep the old code files and called them TenantVariablesProcess.java.4.16.0, so we can easily switch back?

Reply:
I think it's not necessary, as we have the github history.
We can get all the stuff before this change.